### DISCUSSION TITLE
Failed to run npx shadcn@latest add avatar

### DISCUSSION THREAD

**Discussion by YUYING-HUANG2022** [shadcn-ui/ui#5295] [General]
I have already installed  "npx shadcn@latest add datepicker" with "react-day-picker": "8.10.1" installed. 
when I run " npx shadcn@latest add avatar" I got
![屏幕截图 2024-10-11 052205](https://github.com/user-attachments/assets/b284974e-5160-4768-a550-704a7c6a5dd7)
I did use VPN which may not very stable in my area.
but if I try manual way by running "npm install @radix-ui/react-avatar"
Conflicting peer dependency: date-fns@3.6.0
![屏幕截图 2024-10-11 052451](https://github.com/user-attachments/assets/8d334c7c-32f7-4fed-9548-6d170402c8e4)
it appears that two packages related to these two components have a version conflict.
what should I do?

**Comment by Hannes0730** [comment]
I solved this by downgrading the `date-fns` package. 

1. Update package.json by modifying this `"date-fns": "^3.6.0"`
2. Run `npm install`
3. Install the avatar module

Edit: Faced this error when installing the calendar module. Not sure if this will work for your problem.
